src/core.js

```diff
@@ -1232,6 +1233,11 @@ $.extend( $.validator, {
 		range: function( value, element, param ) {
 			return this.optional( element ) || ( value >= param[ 0 ] && value <= param[ 1 ] );
 		},
+		
+		// http://jqueryvalidation.org/step-method/
```

Comment:
this would require a additional docs PR at https://github.com/jzaefferer/validation-content